src/resources/ble_esp32.cc

```diff
@@ -1343,24 +1360,28 @@ int BlePeripheralManagerResource::_on_gap(const BleCallbackScope& scope, struct
   return BLE_ERR_SUCCESS;
 }
 
-static Object* object_to_mbuf(Process* process, Object* object, os_mbuf** result) {
```

Comment:
Split into two functions on the right. One that takes an object and one that takes a blob.